src/ontology/uberon-edit.obo

```diff
@@ -222532,6 +222532,25 @@ synonym: "sagittal ridge" EXACT []
 is_a: UBERON:4200133 ! crest
 relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 
+[Term]
+id: UBERON:7500117
+name: opisthocranion
+def: "A craniometric point which is the furthest point from the glabella." [https://locatelambda.org/manual/, ISBN:9780873659505, Wiktionary:opisthocranion]
+comment: In pigs this point is projected into space as a point between a line connecting the aboral points of the right and left superior nuchal lines. {xref="ISBN:9780873659505"}
```

Comment:
Cannot access the ISBN reference to confirm.
Also, I'm trying to reconcile the comment with the text definition. Is the opisthocranion located on the surface of the skull in all species other than pigs?